docs/roleplay/mastery.md

```diff
@@ -0,0 +1,135 @@
+# Bending Mastery
+
+## Becoming a Master
+
+For a character to become a master of their element they must complete an average of 10 roleplay lessons, minimum 8, depending on their current skill level, age, and ability. A minimum of 8/10 lessons must be completed on the server. 2/10 can be in the [Roleplay Hub](https://discord.gg/tjqX25pH37) if necessary.
+
+## Bending Instructors
+
+### Types of Instructor
+
+There are two types of bending instructor:
+
+- **Masters** have completed all lessons and have an accepted mastery application
+- **Teachers** have completed any amount of the first 5 lessons and can pass their knowledge on to others
```

Comment:
```suggestion
There are two types of bending instructors:

- **Teachers** have completed any amount of the first 5 lessons and can pass this knowledge on to others
- **Masters** have completed all lessons and have an accepted mastery application
```
I've reordered these to be in the order in which people actually earn those roles